inductiva/resources/machines_base.py

```diff
@@ -47,14 +47,12 @@ def __init__(self,
         if machine_type not in inductiva.resources.list_available_machines():
             raise ValueError("Machine type not supported")
 
-        if data_disk_gb < 0 or data_disk_gb > 100:
+        if data_disk_gb < 0 or data_disk_gb > 1940:
```

Comment:
According to the description of the pull request shouldn't we use `data_disk_gb > 20` here?

Reply:
> According to the description of the pull request shouldn't we use `data_disk_gb > 20` here?

We just need to check if data_disk_gb is positive. I fixed that on the wrong branch and ended up forgetting to change it. Will be fixed on the next commit.